src/dto/KeyDto.js

```diff
@@ -0,0 +1,29 @@
+/** Class representing key data. */
+export default class KeyDto {
+    /**
+     * Create new key data.
+     * @param {String} value - The key's value.
+     */
+    constructor(value = '') {
+        this.value = value;
```

Comment:
Why do we need a whole class if there's only one value inside?

Reply:
uhh idk consistency?

Reply:
idk your the database master what should I do?